cdap-ui/Gulpfile.js

```diff
@@ -273,6 +274,10 @@ gulp.task('watch:js:app:tracker', function() {
   return getExtensionBuildPipeline('tracker')
     .pipe(plug.livereload({ port: 35728 }));
 });
+gulp.task('watch:js:app:logviewer', function() {
+  return getExtensionBuildPipeline('logviewer')
+    .pipe(plug.livereload({ port: 35728 }));
```

Comment:
Might want to set this to `35729`? watch might not work for both tracker and logviewer